scripts/vscode-build.ps1

```diff
@@ -14,7 +14,7 @@ $projectFileInfo = Get-ProjectFile $fileInfo
 if ($projectFileInfo) {
   $buildTool = InitializeBuildTool
   $frameworkArg = if ($framework -ne "") { " -p:TargetFramework=$framework" } else { "" }
-  $buildArgs = "$($buildTool.Command) -v:m -m -p:RunAnalyzersDuringBuild=false -p:GenerateFullPaths=true$frameworkArg $($projectFileInfo.FullName)"
+  $buildArgs = "$($buildTool.Command) -v:m -tl:off -m -p:RunAnalyzersDuringBuild=false -p:GenerateFullPaths=true$frameworkArg $($projectFileInfo.FullName)"
```

Comment:
The terminal logger isn't compatible with the VS Code problem matcher.

Reply:
If we preferred the terminal logger, we could write a matcher for it. https://code.visualstudio.com/Docs/editor/tasks#_defining-a-problem-matcher